api/src/main/java/org/openmrs/Order.java

```diff
@@ -807,4 +823,44 @@ public Date getEffectiveStopDate() {
 		return this.getDateStopped() != null ? this.getDateStopped() : this.getAutoExpireDate();
 	}
 	
+	/**
+	 * @since 1.12
+	 * {@link org.openmrs.OrderGroup}
+	 * @returns the OrderGroup
+	 */
+	public OrderGroup getOrderGroup() {
+		return orderGroup;
+	}
+	
+	/**
+	 * @since 1.12
+	 * Sets the OrderGroup for that order.
+	 * If the order is ordered independently, it does not set an orderGroup for it.
+	 * If the order is ordered as an orderGroup, then sets a link to the OrderGroup for that particular order.
+	 * @param orderGroup
+	 */
+	public void setOrderGroup(OrderGroup orderGroup) {
+		this.orderGroup = orderGroup;
+	}
+	
+	/**
+	 * @since 1.12
+	 * Gets the sortWeight for an order if it is ordered as an OrderGroup.
+	 * Doesn't get the sortWeight if the order is not ordered as an OrderGroup.
+	 * @return the sortWeight
+	 */
+	public Double getSortWeight() {
+		return sortWeight;
+	}
+	
+	/**
+	 * @since 1.12
+	 * Sets the sortWeight for an order if it is ordered as an OrderGroup.
+	 * Doesn't set the sortWeight if the order is not ordered as an OrderGroup.
```

Comment:
Should include text like:

> \* \<tt>sortWeight\</tt> is used internally by the API to manage sequencing of orders when grouped and its 
> \* value may be changed as needed by the API for that purpose. Instead of setting this internal API value 
> \* directly, please use {@link OrderGroup#insert(Order, Integer)}.
> \* @see OrderGroup#insert(Order, Integer)

Also, the comment says "Doesn't set the sortWeight if the order is not ordered as an OrderGroup." Not only does the method not do anything to enforce this behavior, I'm not sure how you would unless you changed this method _not_ to change sortWeight if orderGroup is null, but that would mean that order group code would have to ensure orderGroup was set before setting sortWeight (which, if not done correctly, could cause "mysterious" failures to sequence orders). I would suggest just removing this second sentence from the javadoc altogether.